README.md

```diff
@@ -112,6 +112,21 @@ keys:
 
   This integer specifies the LUKS version to use.
 
+- `encryption_clevis_pin`
```

Comment:
the word "pin" implies that this is a PIN (personal identification number) i.e. some sort of password type value.  Should this be `encryption_clevis_type` or `encryption_clevis_algorithm` instead?

Reply:
I used "pin" (both in Blivet and here) because this is what Clevis calls its "plugins": _"In clevis terminology, a pin is a plugin which implements automated decryption."_
I am not against changing the name to `encryption_clevis_type` but `encryption_clevis_pin` might be better for people already familiar with Clevis/Tang.